src/components/Identity/createComponent.js

```diff
@@ -1,27 +1,22 @@
 import createCustomerIds from "./customerIds/createCustomerIds";
-import { defer } from "../../utils";
-import { EXPERIENCE_CLOUD_ID } from "./constants/cookieNames";
-import createMigration from "./createMigration";
+import { defer, cookieJar, getNamespacedCookieName } from "../../utils";
+import { IDENTITY_COOKIE_KEY } from "../../constants/cookieDetails";
 import areThirdPartyCookiesSupported from "../../utils/areThirdPartyCookiesSupported";
 import getBrowser from "../../utils/getBrowser";
 
-const addIdsContext = (payload, ecid) => {
-  payload.addIdentity(EXPERIENCE_CLOUD_ID, {
-    id: ecid
-  });
-};
-
-export default (idSyncs, config, logger, cookieJar, optIn, eventManager) => {
+export default (config, logger, optIn, eventManager) => {
+  const { orgId } = config;
+  const identityCookieName = getNamespacedCookieName(
+    orgId,
+    IDENTITY_COOKIE_KEY
+  );
   let deferredForEcid;
-  let alreadyQueriedForIdSyncs = false;
-  const { idMigrationEnabled, orgId } = config;
-  const migration = createMigration(orgId, idMigrationEnabled);
+  // TODO: Reimplement ID syncs
+  // let alreadyQueriedForIdSyncs = false;
 
   // TODO: Fetch from server if ECID is not available.
   const getEcid = () => {
-    const ecid =
-      cookieJar.get(EXPERIENCE_CLOUD_ID) ||
-      migration.getEcidFromLegacyCookie(cookieJar);
+    const ecid = cookieJar.get(identityCookieName);
```

Comment:
So this will fail, since the `kndctr_orgid_identity`cookie is currently a JAR right?

Reply:
Ah, yeah, I need to change this function from `getEcid` to `hasEcid` and then comment out the `getEcid` command for now.

Reply:
Done.